docs/reference/symbolic.rst

```diff
@@ -0,0 +1,65 @@
+
+.. _symbolic_math:
+
+=============
+Symbolic Math
+=============
+
+You can construct a model, for example a CQM, from symbolic math, which is especially
+useful for learning and testing with small problems.
+
+dimod supports binary and integer variables:
+
+>>> from dimod import Binary, Integer
+>>> # Create binary variables
+>>> x = Binary('x')
+>>> y = Binary('y')
+>>> # Create integer variables
+>>> i = Integer('i')
+>>> j = Integer('j')
+
+Such variables are constructed as either BQMs or QMS, depending on the type of
+variable:
+
+>>> 2*x
+BinaryQuadraticModel({'x': 2.0}, {}, 0.0, 'BINARY')
+>>> 3*i - x
+QuadraticModel({'i': 3.0, 'x': -1.0}, {}, -0.0, {'i': 'INTEGER', 'x': 'BINARY'}, dtype='float64')
+
+You can express mathematical functions on these variables using Python functions such
+as :func:`sum`:
+
+>>> sum([3*i, 2*i])
```

Comment:
this should use `quicksum` #946 once it's merged.

Reply:
I want one for Python functions and then point this one out for performance

Reply:
Given that it is you who will be making the `quicksum` PR, it will also be you updating this as part of that PR :-)

Reply:
Bad news, it's already merged! Beat you to it

Reply:
Added